packages/odyssey-react-mui/src/labs/OdysseyPickers/Picker.tsx

```diff
@@ -108,12 +108,9 @@ export const OptionMetadataComponent = ({
     <OptionDetails odysseyDesignTokens={odysseyDesignTokens}>
       {metaData.map((meta: OptionMetadata, index: number) => {
         const { detailText, icon } = meta;
-
+        const key = `${detailText}-${index}`;
         return (
-          <OptionDetail
-            key={`${index}-${detailText}`}
-            odysseyDesignTokens={odysseyDesignTokens}
-          >
+          <OptionDetail key={key} odysseyDesignTokens={odysseyDesignTokens}>
```

Comment:
Even though this is a computed value, it resolves to a primitive, so it can simply be:
```ts
<OptionDetail key={`${detailText}-${index}`}
```

I'm not seeing a reason to make it a separate 1-time-use variable.